components/apimgt/org.wso2.carbon.apimgt.rest.api.service.catalog/src/main/resources/service-catalog-api.yaml

```diff
@@ -41,12 +113,34 @@ paths:
         execute an exact search for either name or version the parameter should be given inside double quotation.
 
         `Exact search on name`
-            curl -k -X GET 'https://127.0.0.1:9443/api/am/service-catalog/v0/services?name=%22Pet%20Service%22' -H
-            'Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8'
+        ```
+        curl -k -X GET 'https://127.0.0.1:9443/api/am/service-catalog/v0/services?name=%22Pet%20Service%22' -H
+        'Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8'
+        ```
 
         `Matching search on name`
-            curl -k -X GET 'https://127.0.0.1:9443/api/am/service-catalog/v0/services?name=Pet%20Service' -H
-            'Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8'
+        ```
+        curl -k -X GET 'https://127.0.0.1:9443/api/am/service-catalog/v0/services?name=Pet%20Service' -H
+        'Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8'
+        ```
+
+        `Exact search on version`
+        ```
+        curl -k -X GET 'https://127.0.0.1:9443/api/am/service-catalog/v0/services?version=%221.0.0%22' -H
+        'Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8'
+        ```
+
+        `Matching search on name`
+        ```
+        curl -k -X GET 'https://127.0.0.1:9443/api/am/service-catalog/v0/services?version=1.0.0' -H
+        'Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8'
+        ```
+
+        `Get all versions of a given service`
+        ```
+        curl -k -X GET 'https://127.0.0.1:9443/api/am/service-catalog/v0/services?name=%22Pet%20Service%22' -H
+        'Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8'
+        ```
```

Comment:
Do we have any other way to add multiple examples in x-code-samples?

Reply:
Yes, fixed in 6091a00.